api/routes/v1.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var express = require('express');
+
+function initRouter(backend) {
+  var router = express.Router();
+
+  function mockResponse(req, res, next) {
+    res.send('This is a mocked response. Backed service is: ' + backend.status);
+  }
+
+  router.get('/blocks', mockResponse);
+  router.get('/block/:blockHash', mockResponse);
```

Comment:
[Express 4.0's Router](http://expressjs.com/api.html#router) might do what you're looking for.  Totally okay with merging so long as it gets fixed before shipping!